src/components/all-articles.jsx

```diff
@@ -1,26 +1,30 @@
 import { useEffect, useState } from "react";
-import { useContext } from "react";
 import LatestArticlePreview from "./latest-article-preview";
 import { fetchAllArticles } from "../utils/api-calls";
 import Articles from "./articles";
-import Divider from "./divider";
+import Sorter from "./sorter";
+import { sortByCommentCount } from "../utils/sort-by-comment-count";
 
 export default function AllArticles() {
   const [isLoading, setIsLoading] = useState(true);
   const [articles, setArticles] = useState([]);
-
+  const [sortBy, setSortBy] = useState("created_at");
+  const [orderBy, setOrderBy] = useState("desc");
 
   useEffect(() => {
-    console.log("test")
-    fetchAllArticles()
-      .then((articles) => {
-        setArticles(articles);
-        setIsLoading(false);
-      })
-      .catch((err) => {
-        console.log(err);
-      });
-  }, []);
+    if (sortBy !== "comment_count") {
```

Comment:
Is there a particular reason you need to sort comment count differently than other filters?